index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dan's portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section id="skeleton">
+      <header>
+        <h1 id="welcome">Welcome</h1>
+        <a href="#" id="menu">
+          <img src="images/menu.svg" alt="Drop down menu" />
+        </a>
+      </header>
+      <div id="page">
+        <div id="intro">
+          <p id="denis">Hey there. I’m Denis</p>
+          <p id="trade">I’m a software developer</p>
+        </div>
+        <div id="advert">
+          <p>
+            I can help you build a product, feature or website. Look through
+            some of my work and experience! <br />
+            If you like what you see and have a project you need coded, <br />
+            don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="social">
+          <ul>
+            <li>
+              <a href="#"
+                ><img src="/images/social/twitter.svg" alt="Twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="/images/social/github.svg" alt="Github"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="/images/social/linkedin.svg" alt="LinkedIn"
+              /></a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="/images/social/medium.svg" alt="Medium"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="/images/social/angellist.svg" alt="Arangellist"
+              /></a>
+            </li>
+          </ul>
```

Comment:
- [ ] Kindly make sure your social icon works. There is an error in displaying.
- Your design:
<img width="294" alt="Screenshot 2022-03-17 at 11 11 15 AM" src="https://user-images.githubusercontent.com/70269507/158765824-aff8e64d-7146-4ddc-9b25-edd58d4d8cbd.png">

- Figma design:
<img width="294" alt="Screenshot 2022-03-17 at 11 10 43 AM" src="https://user-images.githubusercontent.com/70269507/158765857-d4478563-e8dc-4534-acd3-0e5d835535c5.png">

Reply:
Hey, sja-thadude, even this part for social icons appears fine on my machine.